AsyncPoco.Tests/AsyncPoco.Tests.csproj

```diff
@@ -147,6 +147,13 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
+  <PropertyGroup>
```

Comment:
Adding the Microsoft.SqlServer.Compact Nuget package automatically added this PostBuild event. This is how it manages to work. If the binaries aren't copied, then the test for SqlCe won't run.